tests/src/python/test_qgsserver.py

```diff
@@ -1164,18 +1319,14 @@ def _img_diff_error(self, response, headers, image, max_diff=10, max_size_diff=Q
         test, report = self._img_diff(response, image, max_diff, max_size_diff)
 
         with open(os.path.join(tempfile.gettempdir(), image + "_result.png"), "rb") as rendered_file:
-            encoded_rendered_file = base64.b64encode(rendered_file.read())
```

Comment:
Oh, I did not know that images were available on cdash too, if that's the case, it's fine for me to remove the dump.